### PR TITLE
(feat) /data/docs landing page and discourse integration (DATAPM-785)

### PR DESCRIPTION
**Pull request by grobbie**
## Done

- Add initial /data/docs landing page
- Add integration with charmhub discourse for charmed spark, mongodb and postgres documentation
- (driveby) fix Spark page layout
- Fix branding on Kafka page to align with /data/docs and current Data Fabric branding

## Issue / Card

Fixes DATAPM-785

## Screenshots

image
image

### CODE REVIEW

File: templates/data/docs/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <h2 class="p-card__title p-heading--3">Charmed MongoDB</h2>
+        <p class="p-card__content u-sv3">Access resources for deploying and operating the MongoDB database on IaaS using Charmed MongoDB.</p>
+        <p class="p-card__content"><a href="/data/docs/mongodb-vm" class="p-button--positive">Access docs</a></p>
+      </div>
+    </div>
+  </div>
+</div>
+</section>
+
+<section class="p-strip--light">
```
Comment (anthonydillon) [inline templates/data/docs/index.html:62]:
```suggestion
<section class="p-strip">
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks for the proposal @grobbie. I have a few comments in the code and a QA issue.

## QA
- Visiting the docs pages results in 500 for me on the demo: https://ubuntu-com-12788.demos.haus/data/docs/mongodb-vm

File: templates/data/docs/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  <div class="row">
+    <div class="col7">
```
[the remark this answers is not in the document]
Comment (grobbie) [reply templates/data/docs/index.html]: effected

File: templates/data/spark/index.html [hunk 3]
```diff
@@ -55,19 +55,34 @@ <h2 class="u-sv3">
...
-      <p><a href="https://snapcraft.io/spark-client" class="p-button">Try our Spark solution now</a></p>
+      <p>
+        <div class="col-5 u-vertically-center u-hide--medium u-hide--small u-align--center">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/40d4f11a-servers.svg",
+            alt="",
+            width="218",
+            height="150",
+            hi_def=True,
+            loading="auto|lazy"
```
[the remark this answers is not in the document]
Comment (grobbie) [reply templates/data/spark/index.html]: effected

File: templates/data/spark/index.html [hunk 4]
```diff
@@ -55,19 +55,34 @@ <h2 class="u-sv3">
...
     <div class="col-4">
+      <h2 class="p-heading--4">Expressway operations</h2>
+      <p>Get started fast &mdash; the open source solution is free to use in any environment, in the data centre and on the cloud.</p>
+      <p><a href="/data/docs/spark-k8s" class="p-button">Learn more</a></p>
+    </div>
+    <!--div class="col-4">
       <h2 class="p-heading--4">Expressway operations</h2>
       <p>Rapidly deploy your Apache Spark<sup>&reg;</sup> applications with the integrated, full stack solution from Canonical.</p>
       <p><a href="https://juju.is/" class="p-button">Learn more about our solutions</a></p>
-    </div>
```
[the remark this answers is not in the document]
Comment (grobbie) [reply templates/data/spark/index.html]: this is now done

File: templates/data/docs/index.html (hunk 1 above)
Comment (grobbie) [reply templates/data/docs/index.html]: ok done

**Comment by carkod** [comment]
Because we never had docs pulled from charmhub, I think the Python module is not getting the keys in our deployed environments (prod and demos) @grobbie Can you add the following to the [konf file](https://github.com/canonical/ubuntu.com/blob/main/konf/site.yaml#L28)

```
- name: CHARMHUB_DISCOURSE_API_KEY
    secretKeyRef:
      key: charmhub-api-key
      name: discourse-api

  - name: CHARMHUB_DISCOURSE_API_USERNAME
    secretKeyRef:
      key: charmhub-api-username
      name: discourse-api
```
And that should match with the API KEY variables that you've provided.
Thanks

**Pull request closed without merging**